cloudify/providerdeployment.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright (c) 2018 GigaSpaces Technologies Ltd. All rights reserved
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudify
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"fmt"
+	"os"
+)
+
+
+/*
+Example for deployments info Which need to be used by cloudify provider
+{
+   "deployments":[
+      {
+         "id":"dep-1",
+         "deployment_type":"node",
+         "node_data_data_type":"cloudify.nodes.Kubernetes.Node"
+      },
+      {
+         "id":"dep-2",
+         "deployment_type":"load",
+         "node_data_type":"cloudify.nodes.ApplicationServer.kubernetes.LoadBalancer"
+      }
+   ]
+}
+ */
+
+type DeploymentsInfo struct {
```

Comment:
Run make test, looks like it will show warnings about "No description about public type".

Reply:
@0lvin So what is missing on this ?